src/Stream.php

```diff
@@ -88,6 +89,29 @@ public static function create($body = ''): StreamInterface
         throw new \InvalidArgumentException('First argument to Stream::create() must be a string, resource or StreamInterface.');
     }
 
+    /**
+     * Creates a new PSR-7 stream from path.
+     *
+     * @internal Implements Psr\Http\Message\StreamFactoryInterface::createStreamFromFile for internal use, library users should default to the actual Factory object
+     */
+    public static function createFromFile(string $filename, string $mode = 'r'): StreamInterface
```

Comment:
If we kept this code in the factory and only surrounded the other two `fopen` with `try/catch`, wouldn't achieve the same thing and allow us to not have this method?

Reply:
This same method is used by `moveTo` and `getStream` from `FileUpload`. Because they too create a `Stream` instance from a path.